pylint_odoo/checkers/no_modules.py

```diff
@@ -140,6 +140,11 @@
         'manifest-version-format',
         settings.DESC_DFLT
     ),
+    'C%d07' % settings.BASE_NOMODULE_ID: (
+        'Method required even a translation in line %s',
```

Comment:
You don't need specify the line
The first part of the message shown: `sale_cash_register/models/sale_cash_register.py:158: [C8107(translation-errors), SaleRegisterSession._check_journal_cash] Method required even a translation in line 158`

You could use:
`sale_cash_register/models/sale_cash_register.py:158: [C8107(translation-errors), SaleRegisterSession._check_journal_cash] Method %s required even a translation`